packages/vscode/README.md

```diff
@@ -0,0 +1,53 @@
+# Kubernetes Secrets for Zowe Explorer
+
+## Installing
+
+Use one of the following methods to install the plug-in:
+
+- Install the plug-in to VS Code from the marketplace
+
+  1. On the left most side of the VS Code window in your workspace, click on the extensions icon
```

Comment:
- Step 1 and Step 3 should end with periods.
- Any time you refer to a specific item in the UI, I would capitalize it and bold it: Click on the **Extensions** icon; Click on the Search bar; Click on the Install button. It's just a good way to let the reader know that you're referring to a specific thing on the UI.